### PR TITLE
Import ff-colors globally through webpack

### PR DESCRIPTION
**Pull request by sherbastian**
## Description

making ff-color variables globally available through webpack imports

## Related Issue(s)

#3731 closes
required in #3656 

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] ~~Suitable unit/system level tests have been added and they pass~~   app functionality not impacted
 - [ ] ~~Documentation has been updated~~
    - [ ] ~~Upgrade instructions~~
    - [ ] ~~Configuration details~~
    - [ ] ~~Concepts~~
 - [ ] ~~Changes `flowforge.yml`?~~
    - [ ] ~~Issue/PR raised on `FlowFuse/helm` to update ConfigMap Template~~
    - [ ] ~~Issue/PR raised on `FlowFuse/CloudProject` to update values for Staging/Production~~

## Labels

 - [ ] Backport needed? -> add the `backport` label
 - [ ] Includes a DB migration? -> add the `area:migration` label

### CODE REVIEW

File: config/webpack.config.js [hunk 1]
```diff
@@ -158,7 +164,8 @@ module.exports = function (env, argv) {
             alias: {
                 // Use vue with the runtime compiler (needed for template strings)
                 // To-do: Remove use of template strings, https://github.com/FlowFuse/flowfuse/issues/3290
-                vue: 'vue/dist/vue.esm-bundler.js'
+                vue: 'vue/dist/vue.esm-bundler.js',
+                '@': path.resolve('frontend/src/ui-components')
```
Comment (joepavitt) [inline config/webpack.config.js:168]: think we may want to go to `frontend/src/` for this, then add the `ui-components` into `line 88`. There is a little legacy here with `ui-components` that we need to clean at some point. The TLDR is that it used to be a standalone repository, then it got silly to maintain, so we merged it into `ui-components`. The `@` alias I've traditionally used for `/src`, and that feels cleaner to me.

Comment (sherbastian) [reply config/webpack.config.js:168]: SGTM! I wanted to ask about the ui-components dir, also about the stylesheets directory. I've regularly grouped them into an assets dir, which included images fonts among other things. We can definitely sort them out

Comment (joepavitt) [reply config/webpack.config.js]: Yeah, it's on that list of "nice to haves" that hasn't ever seen the light of day due to other, more urgent, priorities

**Comment by joepavitt** [comment]
We also have ` ff-utility` scss file that would benefit from this too, so have added that. 

I've also done the tedious task of removing all the manual imports of `ff-colors` and `ff-utility` in the relevant places as there is no point loading these all twice, and this is a sensible PR in which to fix that.

**Review by joepavitt — APPROVED** [review]

**Pull request merged**